src/Sulu/Component/Content/Types/Rlp/Strategy/TreeStrategy.php

```diff
@@ -42,6 +42,13 @@ public function __construct(
         );
     }
 
+    public function getChildPart($resourceSegment)
+    {
+        $resourceSegmentParts = explode('/', $resourceSegment);
```

Comment:
Are you sure that's a good idea? I am just trying to do the opposite of `generatePath`, which also doesn't use any `PHPCR` classes. I know that it results in a path at the end, but I would not put that knowledge already in this class. Is a bit like used `dirname` in a wrong way once.